internal/sysfs/poll_unsupported.go

```diff
@@ -0,0 +1,23 @@
+//go:build !linux && !darwin && !windows
```

Comment:
Did we support FreeBSD at some point?

Reply:
We never did for select, but fair point

Reply:
ps generally speaking it is tricky because we only compile on freebsd (in `make check`) because the VM setup for freebsd was really flakey. What this means is that tests don't run against it for things and you can get ENOSYS just like you could for solaris or plan9. Because there is some support for freebsd (e.g. mmap on the compiler side), it is more confused basically.